zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -206,6 +208,31 @@ public void export(final Record<?> record) {
     }
   }
 
+  @Override
+  public void purge() throws Exception {
+    provider.getIndexDescriptors();
+    provider.getIndexTemplateDescriptors();
+    final var client = clientAdapter.getSearchEngineClient();
+
+    //    final var indexMapping = client.getMappings("operate-*,tasklist-*", MappingSource.INDEX);
+    final var indexMapping =
+        client.getMappings("camunda-*,operate-*,tasklist-*", MappingSource.INDEX);
```

Comment:
Are these all the indices we need to delete or are there more?

Reply:
You need to make sure that the indices are prefixed, you can find this in the config.

Reply:
Otherwise it is correct (for now, util we add Optimize)

This overview might help (showing runtime, indices) History indices are not shown, but they follow similar pattern.
![harmonized-indices-miro-v2](https://github.com/user-attachments/assets/42f4d67b-017c-4594-b6d4-9015b4f60d29)